eberron.md

```diff
@@ -113,22 +123,21 @@ When cast at 6th level or above, the weapon gains Reach 2.
 - **Range:** self
 - **Duration:** 10 minutes.
 
-Cast against 2 terrible dice, targeting an unconscious creature. The creature rises in a stupor and are no longer incapacitated. While in this state they keep their temporary wounds and continue to roll death saving throws. They cannot take any action except movement and attacks, and any turn they do not take damage, are attacked, or attack another creature they automatically fail their death saving throw. The spell ends if they regain enough wounds to regain consciousness.
+Cast against 2 terrible dice, targeting an unconscious creature. The creature rises in a stupor and are no longer incapacitated. While in this state they keep their temporary wounds and continue to roll death saving throws. They cannot take any action except movement and attacks, and any turn they do not take damage, are attacked, or attack another creature they automatically fail their death saving throw. While under this effect they ignore the first wound of damage they take from attacks each round. The spell ends if they regain enough wounds to regain consciousness.
 
 For each level cast above 3rd, add an enhanced die.
 
-Reach increases by 1 and you add a white die every level.
-
 #### Ritual of Healing Blood
 *3rd Level*
 
 - **Casting Time:** 9 AP
 - **Range:** 3
 - **Duration:** instant.
+- **Damage Type:** death
 
-Pull the life force from a creature to heal another. Target up to 3 injured and living creatures. Make a spellcasting roll adding a bad die for each creature targeted past the first. Creatures make a stamina save against this attack, taking 2 wound dice in death damage on fail. A target creature in range heals for 1 + half the damage dealt.
+Pull the life force from a creature to heal another. Target up to 5 injured and living creatures. Make a spellcasting roll adding a terrible die for each creature targeted past the first. Creatures make a stamina save against this attack, taking 2 wound dice in death damage on fail. A target creature in range heals for 1 + half the damage dealt.
 
-For each level cast above 3rd, add another 2 wound die in damage.
+For each level cast above 3rd, add an enhanced die
```

Comment:
I'd be very careful with adding wound dice to this spell, if at all. The damage/healing scales very quickly with that many wound dice to the point that is both one of the strongest damage spells while also being the strongest healing spell.